a4_APIweatherApp.py

```diff
@@ -0,0 +1,90 @@
+'''
+This program is inteded to use the Open Meteo API
+and output an API and Mock plot of temperatures in the next 24 hours
+'''
+
+from abc import ABC, abstractmethod
+import requests
+import matplotlib.pyplot as plt
+
+class Service(ABC):
+    @abstractmethod
+    def getting(self):
```

Comment:
The name of the function is is very vague here. What are you 'getting'. When you are writing code, you will often have context in your head, that probably won't make sense to someone else reading your code if the naming of your variables and functions is not explicit. its okay to have longer function names, but much less preferable to have generic names for functions like this.

Maybe rename it as `get_weather_data`?

Check this resource out, this page also has some other best practices which are important to get used to as you are writing more code.

https://realpython.com/python-pep8/#how-to-choose-names